CMakeLists.txt

```diff
@@ -27,10 +27,10 @@ PROJECT(h2o)
 SET(VERSION_MAJOR "2")
 SET(VERSION_MINOR "3")
 SET(VERSION_PATCH "0")
-SET(VERSION_PRERELEASE "-DEV")
+SET(VERSION_PRERELEASE "-beta1")
 SET(VERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}${VERSION_PRERELEASE}")
 SET(LIBRARY_VERSION_MAJOR "0")
-SET(LIBRARY_VERSION_MINOR "14")
+SET(LIBRARY_VERSION_MINOR "15")
```

Comment:
This needs to be set to 16, because we are working for the release _after_ 2.3.0-beta1.